src/executionFiles/bridges/bridge.execute.ts

```diff
@@ -45,6 +63,7 @@ export class BridgeExecutionManager {
     if (!oldCrossProcess || !oldCrossProcess.txHash) {
       if (action.fromToken.address !== constants.AddressZero) {
         // Check Token Approval only if fromToken is not the native token => no approval needed in that case
+        await chainSwitchClosure()
```

Comment:
This will not stop execution here if `!updatedSigner`. Is this on purpose?

Reply:
the chainswitch does that

Reply:
How? It simply returns the execution, but nothing is done with that information as far as I can see?